src/components/hook.js

```diff
@@ -46,6 +46,22 @@ export function createHook(Store, { selector } = {}) {
       }
     };
 
+    const unsubRef = useRef();
+
+    const registerUpdateFn = () => {
```

Comment:
When component init, this will called, and set the `unsubRef`.

And when it goes to `useEffect`, this will get called again but does nothing.